packages/plugin-rdk/src/index.ts

```diff
@@ -1,1404 +1,1415 @@
-/**
- * RDK plugin for JsPsych
- * ----------------------
- *
- * This code was created in the Consciousness and Metacognition Lab at UCLA,
- * under the supervision of Brian Odegaard and Hakwan Lau
- *
- * We would appreciate it if you cited this paper when you use the RDK:
- * Rajananda, S., Lau, H. & Odegaard, B., (2018). A Random-Dot Kinematogram for Web-Based Vision Research. Journal of Open Research Software. 6(1), p.6. DOI: [http://doi.org/10.5334/jors.194]
- *
- * ----------------------
- *
- * Copyright (C) 2017  Sivananda Rajananda
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-import jsPsych from "jspsych";
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
 
-const plugin = <any>{};
-
-plugin.info = {
+const info = <const>{
   name: "rdk",
   parameters: {
+    /* The valid keys that the subject can press to indicate a response. */
     choices: {
-      type: jsPsych.plugins.parameterType.KEY,
+      type: ParameterType.KEYS,
       pretty_name: "Choices",
-      default: jsPsych.ALL_KEYS,
+      default: "ALL_KEYS",
       array: true,
-      description: "The valid keys that the subject can press to indicate a response",
     },
+    /* The correct keys for that trial. */
     correct_choice: {
-      type: jsPsych.plugins.parameterType.KEY,
+      type: ParameterType.KEYS, // TO DO: docs says this can be array or string - need to allow both types here?
```

Comment:
Sounds good to me. This will require some changes to the RDK `trial` method, so I'll do that separately. 

The only other similar case I found is with the `foil` parameter in the `visual-search-circle` plugin - it can be a single image string,  in which case the same image is repeated based on `set_size`, or it can be an array of images, in which case each image is used once. I think we can use the same solution here: get rid of the string option and require an array. If there's only one image in the array then it will be repeated up to `set_size`, otherwise each image in the array is used once. Again, I'll commit this separately.